ubuntu/ubuntu-cloudimg.pkr.hcl

```diff
@@ -128,6 +132,19 @@ source "qemu" "qemu" {
   use_backing_file       = true
 }
 
+build {
+  sources = ["source.null.dependencies"]
+
+  provisioner "shell-local" {
+    inline = [
+      "cp /usr/share/${lookup(local.uefi_imp, var.architecture, "")}/${lookup(local.uefi_imp, var.architecture, "")}_CODE.fd ${lookup(local.uefi_imp, var.architecture, "")}_CODE.fd",
```

Comment:
not needed, but won't hurt

Reply:
Yep, let's remove it not to confuse others of this not needed file copy.